src/test/python/test_signer.py

```diff
@@ -0,0 +1,188 @@
+#   Copyright 2017 Covata Limited or its affiliates
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+import pytest
+from covata.delta import signer
+import json
+import re
+from datetime import datetime
+import six.moves.urllib as urllib
+from covata.delta.signer import __encode_uri, SignatureMaterial
```

Comment:
Refrain importing static functions in favor of namespaces. `__encode_uri` should be accessed via its namespace, i.e. `signer.__encode_uri`. It's okay to import classname, i.e. `from covata.delta.signer import SignatureMaterial` is fine.